opendata_module/opmon_opendata/tests/test_api.py

```diff
@@ -190,6 +192,61 @@ def test_get_harvest_from(db, http_client, caplog):
     assert 'returning 2 rows' in caplog.text
 
 
+def test_get_harvest_from_row_id(db, http_client):
+
+    log_factory(db, request_in_dt='2022-11-07T07:50:00', id=1)
+    log_factory(db, request_in_dt='2022-11-07T08:00:00', id=2)
+
+    # happy logs
+    log_factory(db, request_in_dt='2022-11-07T09:00:00', id=3)
+    log_factory(db, request_in_dt='2022-11-07T10:00:00', id=4)
+    log_factory(db, request_in_dt='2022-11-07T11:00:00', id=5)
+
+    data = {
+        'from_dt': '2022-11-07T07:00:00',
+        'from_row_id': 2,
+    }
+    response = http_client.get('/api/harvest', data)
+    assert response.status_code == 200
+    response_data = response.json()
+    data = response_data.get('data')
+    assert len(data) == 3
+
+
+def test_get_harvest_timestamp_tz(db, http_client):
+    tzinfo = pytz.timezone('Europe/Helsinki')
+    log_factory(db, request_in_dt=datetime.datetime(2022, 11, 7, 7, tzinfo=tzinfo))
+    log_factory(db, request_in_dt=datetime.datetime(2022, 11, 7, 8, tzinfo=tzinfo))
+
+    # happy logs
+    log_factory(db, request_in_dt=datetime.datetime(2022, 11, 7, 9, tzinfo=tzinfo))
+    log_factory(db, request_in_dt=datetime.datetime(2022, 11, 7, 10, tzinfo=tzinfo))
+    log_factory(db, request_in_dt=datetime.datetime(2022, 11, 7, 11, tzinfo=tzinfo))
+
+    data = {
+        'from_dt': '2022-11-07T09:00:00',
+        'timestamp_tz': 'Europe/Helsinki'
+    }
+    response = http_client.get('/api/harvest', data)
+    assert response.status_code == 200
+    response_data = response.json()
+    data = response_data.get('data')
+    assert len(data) == 3
```

Comment:
any assertion on the objects inside the list?

Reply:
Added